actions/st2_chg_ver_for_st2_dockerfiles.sh

```diff
@@ -8,8 +8,10 @@ set -e
 FORK=$1
 PROJECT=$2
 ST2_VERSION=$3
-BRANCH=$4
-LOCAL_REPO=$5
+LOCAL_REPO=$4
+
+SHORT_VERSION=`echo ${ST2_VERSION} | cut -d "." -f1-2`
+BRANCH="v${SHORT_VERSION}"
```

Comment:
I prefer the branch to be passed explicitly into the script.  All of the other st2_cfg_ver_*.sh scripts pass the branch into the script. You can use YAQL expression to get the short version to pass in from the workflow.